arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Resource.kt

```diff
@@ -507,6 +497,7 @@ public sealed class Resource<out A> {
   public companion object {
 
     @PublishedApi
+    @Deprecated("This will be removed from the binary in Arrow 2.0", level = DeprecationLevel.ERROR)
     internal val unit: Resource<Unit> = just(Unit)
```

Comment:
Should we also remove those in Schedule, arrow.fx.coroutines.builder.kt?

Reply:
This was not even being used, but was also not showing up as _unused_ :/ I think any _unused_ internal stuff should be deprecated, and removed.